ReportSimulationOutput/measure.rb

```diff
@@ -475,11 +501,28 @@ def get_timestamps(timeseries_frequency)
     values = @sqlFile.execAndReturnVectorOfString(query)
     fail "Query error: #{query}" unless values.is_initialized
 
+    if timestamps_local_time == 'DST'
+      dst_start_ts = Time.utc(@hpxml.header.sim_calendar_year, @hpxml.header.dst_begin_month, @hpxml.header.dst_begin_day, 2)
+      dst_end_ts = Time.utc(@hpxml.header.sim_calendar_year, @hpxml.header.dst_end_month, @hpxml.header.dst_end_day, 1)
+    elsif timestamps_local_time == 'UTC'
+      utc_offset = @hpxml.header.time_zone_utc_offset
+      utc_offset *= 3600 # seconds
+    end
+
     timestamps = []
     values.get.each do |value|
       year, month, day, hour, minute = value.split(' ')
       ts = Time.utc(year, month, day, hour, minute)
-      timestamps << ts.strftime('%Y/%m/%d %H:%M:00')
+
+      if timestamps_local_time == 'DST'
+        if (ts >= dst_start_ts) && (ts < dst_end_ts)
+          ts += 3600 # 1 hr shift forward
+        end
+      elsif timestamps_local_time == 'UTC'
+        ts -= utc_offset
+      end
+
+      timestamps << ts.iso8601
```

Comment:
@nmerket suggests that there should only be a "Z" at the end of the time string for the TimeUTC column.

![image](https://user-images.githubusercontent.com/5861765/151068748-3257f322-c373-40ae-b9d3-1edf840436a6.png)

Reply:
`$ /c/openstudio-3.3.0/bin/openstudio.exe workflow/run_simulation.rb -x workflow/sample_files/base.xml --hourly ALL --add-timeseries-time-column DST --add-timeseries-time-column UTC`:

![image](https://user-images.githubusercontent.com/8516790/151082815-32679e86-1b63-4151-8853-87262bdf31db.png)

Reply:
?

Reply:
I misinterpreted what you wrote. How about this?

![image](https://user-images.githubusercontent.com/8516790/151085208-fe603c59-d97e-4761-8046-d7aecbac7cd4.png)

Reply:
Yes, that's what we meant. Thanks!